opentracing_instrumentation/local_span.py

```diff
@@ -24,6 +24,16 @@
 import tornado.concurrent
 from . import get_current_span, span_in_stack_context, utils
 
+try:
+    import asyncio
+    _ASYNCIO = True
+except Exception:
```

Comment:
I think it's better to use a specific exception here.

```suggestion
except ImportError:
```